flamedisx/xenon/x1t_sr1.py

```diff
@@ -305,10 +305,16 @@ def electron_gain_std(s2_relative_ly,
         # 0 * light yield is to fix the shape
         return single_electron_width + 0. * s2_relative_ly
 
+    @staticmethod
+    def double_pe_fraction(z, *, dpe=DEFAULT_P_DPE):
+        # Ties the double_pe_fraction model function to the dpe
+        # parameter in the sources
+        return dpe + 0 * z
+
     #TODO: implement better the double_pe_fraction or photon_detection_efficiency as parameter
```

Comment:
This comment can be removed

Reply:
As discussed, this comment is a relic from the flamedisx before the DPE block was added so it is indeed ok to remove this comment.